recipes/jsoncpp/all/conanfile.py

```diff
@@ -31,9 +35,7 @@ def configure(self):
             del self.options.fPIC
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = "jsoncpp-{}".format(self.version)
-        os.rename(extracted_dir, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
```

Comment:
Could you add `required_conan_version = ">=1.33.0"` please?

Reply:
Added a [hook PR](https://github.com/conan-io/hooks/pull/368) to remind about it :)